tests/tests_integration/test_api/test_datapoint_subscriptions.py

```diff
@@ -148,6 +149,24 @@ def test_update_subscription(self, cognite_client: CogniteClient, time_series_ex
             assert updated.time_series_count == len(time_series_external_ids) - 1
             assert updated.data_set_id == data_set.id
 
+    def test_update_subscription_write_object(self, cognite_client: CogniteClient, time_series_external_ids: list[str]):
+        new_subscription = DataPointSubscriptionWrite(
+            external_id=f"PYSDKDataPointSubscriptionUpdateTest-{random_string(10)}-{RUN_UNIQUE_ID}",
```

Comment:
Isn't the the run_unique_id redundant with the random string?

Reply:
Agreed, a bit overkill.